ocs_ci/framework/pytest_customization/ocscilib.py

```diff
@@ -421,3 +432,58 @@ def pytest_runtest_makereport(item, call):
             collect_performance_stats(test_case_name)
         except Exception as ex:
             log.error(f"Failed to collect performance stats. Error: {ex}")
+
+
+def set_report_portal_tags(config):
+    rp_tags = list()
+    rp_tags.append(ocsci_config.ENV_DATA.get('platform'))
+    rp_tags.append(ocsci_config.ENV_DATA.get('deployment_type'))
+    if ocsci_config.REPORTING.get('us_ds') == 'us':
+        rp_tags.append('upstream')
+    else:
+        rp_tags.append('downstream')
+    worker_instance_type = ocsci_config.ENV_DATA.get('worker_instance_type')
+    rp_tags.append(f"worker_instance_type:{worker_instance_type}")
+    rp_tags.append(f"ocp_version:{get_ocp_version()}")
+    rp_tags.append(
+        f"ocs_version:{ocsci_config.ENV_DATA.get('ocs_version')}"
+    )
+    if ocsci_config.DEPLOYMENT.get('ocs_registry_image'):
+        ocs_registry_image = ocsci_config.DEPLOYMENT.get('ocs_registry_image')
+        rp_tags.append(f"ocs_registry_image:{ocs_registry_image}")
+        rp_tags.append(f"ocs_registry_tag:{ocs_registry_image.split(':')[1]}")
+    if ocsci_config.DEPLOYMENT.get('ui_deployment'):
+        rp_tags.append('ui_deployment')
+    if ocsci_config.DEPLOYMENT.get('live_deployment'):
+        rp_tags.append('live_deployment')
+    if ocsci_config.DEPLOYMENT.get('stage'):
+        rp_tags.append('stage_deployment')
+    if not ocsci_config.DEPLOYMENT.get('allow_lower_instance_requirements'):
+        rp_tags.append("production")
+
+    for tag in rp_tags:
+        if tag:
+            config.addinivalue_line("rp_launch_tags", tag.lower())
+
+
+def get_ocs_csv_version():
```

Comment:
How about moving to utils like get_ocp_version?

Reply:
I originally had this in utils but due to some circular import issues that I didn't want to address in this PR i opted to have it here for now. I can try to move it to utils and fix the import issues but there were several that stemmed from us importing test helpers into our framework code which we shouldn't be doing.

Reply:
👍  I agree on the circular import issues in utils, I am hitting that in my other PR as well

Reply:
I messed around with this and determined I can move it to `ocs_ci.ocs.resources.ocs`. I should have this PR updated shortly with this change.